README.md

```diff
@@ -135,11 +135,34 @@ yabs:run_default_task()
 yabs.run_command('echo hello, world', 'quickfix', { open_on_run = 'always' })
 ```
 
-### ".yabs" files
+### `.yabs` files
 
-The first time you execute `yabs:run_task()`, yabs will look for a file named .yabs in
-the current working directory. If found, it will be sourced as a lua file. This
-is useful for project-local configurations.
+You can create project-local configurations by creating `.yabs` file
+in the project working directory. It will be sourced as a lua file the
+first time you execute `yabs:run_task()`. The file should return a
+table with additional task that will be append to your global
+configuration. The syntax is the same as for [setup()](#setup):
```

Comment:
> append to your global configuration

More like "extend" your configuration. I think it should also be clear that anything you put in the .yabs file will override any already-defined values. It should be obvious, but I think it's best to err on the side of caution here.

Also, the word "global" config could be confusing because on other places we make a distinction between the global config and the filetype specific config, global in that case meaning tasks that apply to every file in the project. Maybe calling it your "main" config or something like that instead would be better?

Reply:
Great, thanks!

I still think it would be more clear if we said "extend your main configuration, overriding any values already defined there". I could see that being a source of confusion otherwise.

Reply:
> extend your main configuration, overriding any values already defined there

English is not my native language, but I would have thought that `.yabs` would replace all values ​​in the main config. What do you think?

Reply:
I guess it's fine...If we get any issues about it then we can always change it later I suppose.

Reply:
Okay, done!